packages/terra-action-header/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -10,9 +10,15 @@
   --terra-action-header-color: #b2b5b6;
 
```

Comment:
Did we miss adding the theme variables for the lowlight theme?

Reply:
No - lowlight is intentionally a partial theme that only supplies new colors, all dimensions and layout values pass thru from default theme by design.